conans/test/unittests/client/tools/system_pm_test.py

```diff
@@ -165,7 +165,7 @@ def system_package_tool_test(self):
             spt.install("a_package", force=False)
             self.assertEqual(runner.command_called, "rpm -q a_package")
             spt.install("a_package", force=True)
-            self.assertEqual(runner.command_called, "sudo -A yum install -y a_package")
+            self.assertEqual(runner.command_called, "sudo -A dnf install -y a_package")
```

Comment:
I would say the test is failing because it is not modeled in redhat to use dnf. Should it?

Reply:
Since I'm now only considering Fedora now, it probably shouldn't.